app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    comm = command.split()
+    new_dir = comm[2][:comm[2].rfind("/")]
```

Comment:
For obtainting directory you can use `os.path.dirname`

Reply:
Thank you!